final_task/rss_reader/classes/news_class.py

```diff
@@ -22,17 +22,34 @@ def print_feed_title(self):
         """Print feed title"""
         print(self.feed_title)
 
+    def create_list_of_links(self):
+        """Turn tuple ([href_links], [img_links]) into list of links"""
+        # Merge list of links
+        list_of_links = []
+        list_of_links.extend(self.links[0])
+        list_of_links.extend(self.links[1])
+        return list_of_links
```

Comment:
A question:

If you absolutely sure that `self.links` corresponds to specified structure, why not use something like `self.links[0] + self.links[1]` instead of this function.

If there is possibility that `self.links` will have more than 2 elements, then it is better to use cycle and append to form a list.

Reply:
It was conceived, that only two elements in this structure. Class method create_string_repr_of_links confused me.

Reply:
Fixed